src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2423,25 +2435,25 @@ namespace HVACUnitarySystem {
                                                      UnitarySystem(UnitarySysNum).CoolingCoilName, ErrFound);
             EqSizing.DesCoolingLoad = DXCoolCap;
 
-            for (Iter = 1; Iter <= UnitarySystem(UnitarySysNum).NumOfSpeedCooling; ++Iter) {
+            for (Iter = UnitarySystem(UnitarySysNum).NumOfSpeedCooling; Iter >= 1;
+                --Iter) { // use reverse order since we divide by CoolVolumeFlowRate(max)
```

Comment:
Gost to go backwards since it's a divide by UnitarySystem(UnitarySysNum).CoolVolumeFlowRate(Iter)